__tests__/api/BundleDeploy/BundleDeployer.test.ts

```diff
@@ -200,6 +200,44 @@ describe("BundleDeployer01", () => {
                                   "//         MSGCLASS=X,TIME=NOLIMIT";
         await testDeployJCL(parms);
     });
+    it("should support long line jobcard", async () => {
+
+        let parms: IHandlerParameters;
+        parms = DEFAULT_PARAMTERS;
+        setCommonParmsForDeployTests(parms);
+        parms.arguments.resgroup = "12345678";
+        parms.arguments.jobcard = "//DFHDPLOY JOB DFHDPLOY,CLASS=A,MSGCLASS=X,TIME=NOLIMIT,NOEL=ABCDEFGHIJKMNOPQRSTUVWXYZ";
+        await testDeployJCL(parms);
```

Comment:
Is it worth doing a bit more validation here to check the jobcard in the generated JCL is broken up as you expect?

Reply:
That's what the snapshot output confirms. It took a few goes to get it right, if it should ever change in the future then the snapshots will no longer match.